package.json

```diff
@@ -52,6 +54,7 @@
     "eslint-plugin-jsx-a11y": "^6.5.1",
     "eslint-plugin-react": "^7.27.1",
     "eslint-plugin-react-hooks": "^4.3.0",
+    "redux": "^4.1.2",
```

Comment:
- [ ] Also update the readme.md file following [this template](https://github.com/microverseinc/readme-template).